source/MaterialXCore/CMakeLists.txt

```diff
@@ -27,14 +27,16 @@ target_include_directories(MaterialXCore
     PRIVATE
         ${EXTERNAL_INCLUDE_DIRS})
 
-install(TARGETS MaterialXCore
-    EXPORT MaterialX
-    ARCHIVE DESTINATION ${MATERIALX_INSTALL_LIB_PATH}
-    LIBRARY DESTINATION ${MATERIALX_INSTALL_LIB_PATH}
-    RUNTIME DESTINATION bin)
-
-install(FILES ${materialx_headers}
-    DESTINATION ${MATERIALX_INSTALL_INCLUDE_PATH}/MaterialXCore/)
-
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_BUILD_TYPE}/MaterialXCore.pdb"
-    DESTINATION "${MATERIALX_INSTALL_LIB_PATH}/" OPTIONAL)
+if(NOT SKBUILD)
```

Comment:
Because we statically link, we don't need and want to install the libraries. They would end up in the package which is useless and adds weight.